tests/lib/lib.d.ts

```diff
@@ -1,14 +1,14 @@
 /*! *****************************************************************************

```

Comment:
do not change this file

Reply:
I think that was automatically generated